prod-devel-rcar-s4.yaml

```diff
@@ -44,9 +44,6 @@ common_data:
 
   # Common configuration options for all yocto-based domains
   conf: &COMMON_CONF
-    - [SSTATE_DIR, "${TOPDIR}/../../../common_data/sstate"]
-    - [DL_DIR, "${TOPDIR}/../../../common_data/downloads"]
```

Comment:
I can understand why sstate can be buggy. But do we really need to separate downloads?

Reply:
You are right, I will update the commit.

Also, I will add more commits to this PR to address comments from the external mail.

So this PR is draft and [WIP] until testing ends.